rts/lra/client/pom.xml

```diff
@@ -29,6 +29,16 @@
       <artifactId>lra-service-base</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
+      <version>${version.org.jboss.resteasy}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-jackson2-provider</artifactId>
+      <version>${version.org.jboss.resteasy}</version>
+    </dependency>
```

Comment:
After a chat with @ruromero, we can confirm that the `resteasy-jackson2-provider` dependency will work nicely with Quarkus based applications :+1:
Thanks Ruben